third_party/lua/luadebug.lua

```diff
@@ -538,6 +751,9 @@ local function cmd_quit()
 end
 
 local commands_help = {
```

Comment:
The debugger aborts with error on a command with tipo:
```
luadebug.lua: Loaded for 2.11.0-entrypoint-917-g1c6abdd76
break via dbg() => ./main.lua:8 in chunk at ./main.lua:0
   5   
   6   dbg()
   7   
   8=> hello_1()
   9   hello_2()
  10   
  11   local compat = require('tarantool').compat
luadebug.lua> hel;
INTERNAL luadebug.lua ERROR. ABORTING
: builtin/luadebug.lua:117: bad argument #3 to 'format' (no value)
LuajitError: builtin/luadebug.lua:927: INTERNAL luadebug.lua ERROR. ABORTING
: builtin/luadebug.lua:117: bad argument #3 to 'format' (no value)
fatal error, exiting the event loop
```

Reply:
That recently introduced bug was a recent conflict resolution artefact, and fix is quite simple:
```diff
diff --git a/third_party/lua/luadebug.lua b/third_party/lua/luadebug.lua
index 3a158fa0d..89b2eb3c7 100644
--- a/third_party/lua/luadebug.lua
+++ b/third_party/lua/luadebug.lua
@@ -865,7 +865,6 @@ local function run_command(line)
         return unpack({ cmd_eval(line) })
     else
         dbg_write_error("command '%s' is not recognized.\n" ..
-                        "'%s'\n" ..
                         "Type 'h' and press return for a command list.",
                         line)
         return false
```

Thanks!